### DISCUSSION TITLE
Omnisearch Companion

### DISCUSSION THREAD

**Discussion by ALegendsTale** [scambier/obsidian-omnisearch#353] [Show and tell]
Hello,

I’ve benefited greatly from the Omnisearch plugin, particularly the HTTP server and injected search results. Inspired by this, I’ve developed a companion browser extension for both Firefox and Chrome named Omnisearch Companion. It’s designed to use the Omnisearch server to offer browser-wide search beyond what the userscripts cover.

I hope the extension can be a valuable tool for the community. If it seems like a good fit, I would be honored if a link was added to it in the Omnisearch Documentation to help others discover it.

Thanks

• [Repository](https://github.com/ALegendsTale/omnisearch-companion) • [Firefox](https://addons.mozilla.org/en-US/firefox/addon/omnisearch-companion/) • [Chrome](https://chromewebstore.google.com/detail/omnisearch-companion/kcjcnnlpfbilodfnnkpioijobpjhokkd)

**Comment by scambier** [comment]
Thank you, I'll add a link in the readme! You should also probably post this on https://forum.obsidian.md/, and maybe their Discord server if you have an account :)

**Reply by ALegendsTale** [reply]
That's a great idea, thank you! :)
